### PR TITLE
ZEPPELIN-3344. Revert comments in queries in JDBC interpreter

### PR DESCRIPTION
**Pull request by mebelousov**
### What is this PR for?
The original purpose of https://github.com/apache/zeppelin/pull/2158 was correct processing of ';'. This was done via full removing comments from code.
Unfortunately Apache Phoenix uses hooks in comments https://forcedotcom.github.io/phoenix/#hintml.
Thus we should not delete comments in scripts.
There was discussion about comment rules for different databases (solr). The right way is keep style. Thus analysts can copy queries to another tool and can get same results and errors.

### What type of PR is it?
[Bug Fix]

### What is the Jira issue?
* https://issues.apache.org/jira/browse/ZEPPELIN-3344

### How should this be tested?
* Unit tests pass: testSplitSqlQueryWithComments and testSplitSqlQuery

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

File: jdbc/src/test/java/org/apache/zeppelin/jdbc/JDBCInterpreterTest.java [hunk 1]
```diff
@@ -534,7 +534,7 @@ public void testPrecodeWithAnotherPrefix() throws SQLException, IOException {
   }
 
   @Test
-  public void testExcludingComments() throws SQLException, IOException {
+  public void testSplitSqlQueryWithComments() throws SQLException, IOException {
```
Comment (zjffdu) [inline jdbc/src/test/java/org/apache/zeppelin/jdbc/JDBCInterpreterTest.java]: No test case needs to add or change ?

Comment (mebelousov) [reply jdbc/src/test/java/org/apache/zeppelin/jdbc/JDBCInterpreterTest.java]: @zjffdu done.
The tests are green.

**Comment by zjffdu** [comment]
ping @mebelousov do you have chance to look at this comment https://github.com/apache/zeppelin/pull/2876#discussion_r175098216

File: jdbc/src/test/java/org/apache/zeppelin/jdbc/JDBCInterpreterTest.java [hunk 2]
```diff
@@ -189,13 +189,16 @@ public void testSplitSqlQuery() throws SQLException, IOException {
         "select '\n', ';';" +
         "select replace('A\\;B', '\\', 'text');" +
         "select '\\', ';';" +
-        "select '''', ';'";
+        "select '''', ';';" +
+        "select /*+ scan */ * from test_table;" +
+        "--singleLineComment\nselect * from test_table";
+
```
Comment (mebelousov) [inline jdbc/src/test/java/org/apache/zeppelin/jdbc/JDBCInterpreterTest.java]: @zjffdu The last two lines are new tests for processing of the comments.

**Comment by zjffdu** [comment]
Thanks @mebelousov LGTM

**Pull request closed without merging**